tests/serialized_test_data_generation/configs/c96_54ranks_baroclinic.yml

```diff
@@ -0,0 +1,327 @@
+data_table: default
```

Comment:
before this is merged in we should make sure PR 118 is merged to master or here and also generate the c96_54ranks_baroclinic data (to make sure that won't crash jenkins every time that runs). Depending on how long this one takes, we might want to change the frequency of the jenkins validation. But the c96_6ranks was pretty quick.